packages/subgraph/package.json

```diff
@@ -18,7 +18,7 @@
     "dev-deploy-only-rinkey": "yarn prepare-rinkeby && yarn codegen && yarn build && graph deploy psparacino/smart-invoices-rinkey-ps --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ --debug",
     "deploy-only-xdai": "graph deploy psparacino/v1-xdai-smart-invoices --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ --debug",
```

Comment:
Just a heads up for whenever the subgraph is deployed on Gnosis Chain, this is the most recent one: geovgy/v1-gnosis-chain-smart-invoice